src/containers/Auth/ConfirmOTP/ConfirmOTP.tsx

```diff
@@ -1,31 +1,128 @@
 import React, { useState, useContext } from 'react';
 import styles from './ConfirmOTP.module.css';
-import { Typography, FormHelperText } from '@material-ui/core';
+import { Typography, FormHelperText, makeStyles, createStyles } from '@material-ui/core';
 import InputLabel from '@material-ui/core/InputLabel';
 import FormControl from '@material-ui/core/FormControl';
 import OutlinedInput from '@material-ui/core/OutlinedInput';
 import { Button } from '../../../components/UI/Form/Button/Button';
 import clsx from 'clsx';
 import axios from 'axios';
-import { REACT_APP_GLIFIC_REGISTRATION_API } from '../../../common/constants';
+import {
+  REACT_APP_GLIFIC_REGISTRATION_API,
+  REACT_APP_GLIFIC_AUTHENTICATION_API,
+} from '../../../common/constants';
 import { Redirect } from 'react-router-dom';
 import { SessionContext } from '../../../context/session';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { ThemeProvider } from '@material-ui/styles';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import RefreshIcon from '@material-ui/icons/Refresh';
 
 export interface ConfirmOTPProps {
   location: any;
 }
 
+const useStyles = makeStyles(() =>
+  createStyles({
+    continueButton: {
+      width: '330px',
+      borderRadius: '27px',
+      marginTop: '0px',
+    },
+    inputField: {
+      lineHeight: '32px',
+    },
+    titleText: {
+      fontWeight: 'bold',
+      marginBottom: '10px',
+    },
+  })
+);
+
 export const ConfirmOTP: React.SFC<ConfirmOTPProps> = (props) => {
   const { setAuthenticated } = useContext(SessionContext);
   const [userAuthCode, setUserAuthCode] = useState('');
   const [tokenResponse, setTokenResponse] = useState('');
   const [authError, setAuthError] = useState(false);
   const [alreadyExists, setAlreadyExists] = useState(false);
+  const classes = useStyles();
+
+  const theme = createMuiTheme({
```

Comment:
Why are we defining new theme overrides here?  Does not seems right. Can you please evaluate another option.